core/src/main/java/io/questdb/cairo/SecurityContext.java

```diff
@@ -124,6 +123,8 @@ default void clear() {
         // no-op
     }
 
+    void failIfReadOnly();
```

Comment:
lets remove this

Reply:
turned it into:
```
boolean isQueryCancellationAllowed();
```

this is much more descriptive now.